public/locales/en/topic-overview.json

```diff
@@ -1,8 +1,30 @@
 {
-    "pageTitle": "Topics",
-    "pageSubtitle": "A subtitle to talk about an over view of topics and what we hope to do with these. Lorem ipsum dolor sit amet, consectetur.",
-    "topHeading": "Culture and Society",
-    "botHeading": "Church and Missions",
-    "cultureTopics": ["Popular-Religious-Practices", "Ancestor-Veneration", "Harmony", "Social-Conformity"],
-    "churchTopics": ["Church-Leadership", "Personal-Evangelism", "Missionary-Life", "Japanese-Overseas", "Unity-In-Christ"]
+    "pageTitle": "Prayer topics",
+    "pageSubtitle": "After surveying missionaries, pastors, and Christian leaders in Japan, we collected over 500 prayer points and grouped them into topics for Japanese culture and society, and church and mission. The 30 topics chosen were those prioritized by Japanese leaders as key opportunities and challenges for the gospel in Japan today.",
+    "topHeading": "Culture and society",
+    "botHeading": "Church and mission",
+    "cultureLinks": [
```

Comment:
I think this needs some reworking? Maybe in the naming or something? Right now, when I first look at this, links and topics seem identical, which suggests we don't need one or the other?
On the other hand, from the naming, I would expect topics to be something like "Popular Religious Practices" and links to be "popular-religious-practices". 
Also, in TopicOverviewSection, it seems like you use `topics` (used as `labels`) for: key, the Link href (as lowercase), p (with .split/.join formatting) and `links` just for the Image src? 
I'm on the fence on whether I'd prefer combing links and topics as one map, or keeping them separate but have a more clear division of responsibility, but we should probably tidy it up